src/views/TournamentDetail.vue

```diff
@@ -45,8 +48,8 @@ try {
 </script>
 
 <template>
-  <div v-if="tournament?.is_announced" class="min-h-[calc(100vh_-_6rem)]">
-    <div class="sticky top-24 min-w-full bg-[#2c292d]">
+  <div v-if="tournament?.is_announced" class="flex min-h-[calc(100vh_-_6rem)] flex-col">
```

Comment:
This looks better on the info view but unfortunately there is another problem. On the register component, when there is an error with the password validation, the image grow bigger or shrink because of the page height changing.
![image](https://github.com/user-attachments/assets/94afb10d-2090-4f5b-a885-54c90ecea2d6)
![image](https://github.com/user-attachments/assets/96e15a1f-b394-4a04-b252-4586f2e59bf4)

This can be fixed by setting the field validation component to absolute so it doesn't impact the pages.